doc/reference/configuration/configuration_reference.rst

```diff
@@ -2836,6 +2836,38 @@ Attributes must be ``key: value`` pairs with string keys and values.
 
     See also: :ref:`configuration_labels`
 
+..  _configuration_reference_isolated_mode:
```

Comment:
Forgot to say it on the previous review iteration: AFAIR, options/sections on each level are sorted alphabetically. Here `isolated` goes after `labels`.